package.yaml

```diff
@@ -16,15 +16,15 @@ copyright: ! 'Copyright (C) 2010-2013 Tony Morris
   Copyright (C) 2016-2017 Data61'
 license: BSD3
 license-file: etc/LICENCE
-homepage: https://github.com/data61/fp-course
-git: git@github.com:data61/fp-course.git
-bug-reports: https://github.com/data61/fp-course/issues
+homepage: https://github.com/bitemyapp/fp-course
+git: git@github.com:bitemyapp/fp-course.git
+bug-reports: https://github.com/bitemyapp/fp-course/issues
 
 tested-with: GHC==8.0.2
 
 extra-source-files:
 - etc/CONTRIBUTORS
-- changelog
+- CHANGELOG.md
```

Comment:
typo i spose

Reply:
My bad, I renamed the file and forgot to edit that part.